api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -228,3 +234,55 @@ message Runtime {
   // encoding <config_runtime_proto_json>`.
   google.protobuf.Struct base = 4;
 }
+
+message RuntimeLayer {
+  // :ref:`Static runtime <config_runtime_bootstrap>` layer.
+  message StaticLayer {
+    // This follows the :ref:`runtime protobuf JSON representation encoding
+    // <config_runtime_proto_json>`.
+    google.protobuf.Struct value = 1;
+  }
+
+  // :ref:`Disk runtime <config_runtime_local_disk>` layer.
+  message DiskLayer {
+    // Specifies the subdirectory to load within the :ref:`root directory
+    // <envoy_api_field_config.bootstrap.v2.LayeredRuntime.symlink_root>`. This
+    // is useful if multiple systems share the same delivery mechanism. Envoy
+    // configuration elements can be contained in a dedicated subdirectory.
+    string subdirectory = 1;
+
+    // :ref:`Append <config_runtime_local_disk_overrides>` the service cluster
+    // to the path.
+    bool append_service_cluster = 2;
```

Comment:
Is this new or did we already have this? If existing I'm wondering if we can get rid of it in the new API? If new wondering what the reasoning is? If we keep it we should probably flesh out the docs a bit more.

Reply:
Yes, this existed before. The `ovverride_subdirectory` had this quirky behavior where it would append the local service cluster to the path, see https://www.envoyproxy.io/docs/envoy/latest/configuration/runtime#local-disk-file-system. I made this generalized to all layers via this option, which also helped preserve backwards compat. I don't think we can get rid of this without potentially breaking users.

Reply:
OK thanks for refreshing my memory. I think this remains useful so sure let's keep it. Maybe just break out that portion of the docs into its own sub-section and ref-link directly to it? It mainly caught me off guard.